packages/dataviews/src/dataviews-layouts/table/column-header-menu.tsx

```diff
@@ -255,7 +255,7 @@ const _HeaderMenu = forwardRef( function HeaderMenu< Item >(
 									</Menu.ItemLabel>
 								</Menu.Item>
 							) }
-							{ canInsertLeft && (
+							{ canInsertLeft && !! hiddenFields.length && (
```

Comment:
Can we make it disabled if there are no fields to add?

There's a difference between a menu item that we should never display (e.g., never display insert left in the primary column, or never display left/right if the consumer has disabled moving) vs. a situational context (it's possible to insert fields, but there's none at the moment).

Going to tag @jameskoster for thoughts, but I'm approving in the meantime. It can be addressed as follow-up, if we want to implement what I suggested.